app/constants/urls.constants.ts

```diff
@@ -3,7 +3,7 @@ export const checkUsername = (HOST: string, username: string) =>
 
```

Comment:
Are we using getUserSelfData and CalendarId call now?

Reply:
@harshith-venkatesh they are not used as of now but we might need them in future so I haven't removed them.